README.md

```diff
@@ -1 +1,59 @@
-Vesper rewarder contract for Sushi 2x rewards. 
+# Vesper Rewarder
+
+This repository contains the Vesper rewarder contract for Sushi 2x rewards.
+
+## Overview
+
+The `VesperRewarder` contract implements double incentive mechanism for SushiSwap farm feature.
+Whenever a user interacts with the `MasterChefV2` contract (that handle 2x rewards farming pools), the `VesperRewarder.onSushiReward()` function is called, and, that function calculates the VSP reward amount based on how much SLP token he staked and send it to the user.
+
+See more: https://dev.sushi.com/sushiswap/contracts/masterchefv2/adding-double-incentives
+
+## Install
```

Comment:
Since you have a .nvmrc file and specify a version, I think readme should mention the `nvm use` step here.

Reply:
Done.